src/neighbors.cc

```diff
@@ -1275,32 +1275,35 @@ peerRefreshDNS(void *data)
     eventAddIsh("peerRefreshDNS", peerRefreshDNS, nullptr, 3600.0, 1);
 }
 
-static void
-peerConnectFailedSilent(CachePeer * p)
+// TODO: Move to CachePeer::noteFailure() or similar.
+// TODO: Require callers to detail failures instead of using one (and often
+// misleading!) "TCP" label for all of them.
```

Comment:
We will convert this function into a method when improving reporting of the "TCP connection ... failed" problems. That work deserves a dedicated PR.